tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+import sys
+
+sys.path.append('transparency-in-coverage/python')
```

Comment:
In a python project, you want to define your "sources roots." How python builds modules can be ambiguous.